### ISSUE TITLE
False error about :require-macros

### ISSUE THREAD

**Issue by kwladyka** [AtomLinter/linter-clojure#41]
I get an error

<img width="398" alt="screen shot 2016-05-19 at 11 49 03" src="https://cloud.githubusercontent.com/assets/3903726/15389522/b81e2132-1db7-11e6-86cb-f367ced06a97.png">

```clojure
(ns view.product-box
  (:require-macros [cljs.core.async.macros :refer [go]])
  (:require
    [rum.core :as rum :include-macros true]
    [clj-di.core :as di :refer-macros [let-deps]]
    [cljs.core.async :as async :refer [<!]]
    [view.core :as v]
    [core.util :as u]
    [view.ui :as ui]
    [sablono.core :refer-macros [html]]
    [clojure.string :as string]))
```

This code is valid.

**Comment by Arcanemagus** [comment]
If you run the Clojure jar manually on the file does it show the same issue?

**Comment by kwladyka** [comment]
Yes. I guess it is because of `cljs` file, it is ClojureScript and it has some differences https://github.com/clojure/clojurescript/wiki/Differences-from-Clojure. But your module active in that type of files, so i understand you support this files too. Or not and it shouldn't be active in `cljs` files?

**Comment by Arcanemagus** [comment]
I don't use Clojure myself, @trevershick _might_ be able to answer you, but this plugin currently has no active maintainer.

**Comment by Arcanemagus** [comment]
Some of this _might_ have been fixed in #58, when the next version comes out can you test if this is still an issue? The classpath stuff is likely to still be a problem, unfortunately I don't know the Java ecosystem well enough to have a good solution to that.

**Comment by kwladyka** [comment]
not active
